### PR TITLE
Add gen3 6dof controller config and launch instructions

### PR DESCRIPTION
**Pull request by madan96**
This adds the controller yaml file for gen3 6dof arm and also updates the `default.launch` file with additional arguments and node required for `kortex_hardware`.
***

**Before creating a pull request**

- [ N/A ] Document new methods and classes
- [ N/A ] Format code with `make format`

**Before merging a pull request**

- [ N/A ] Add unit test(s) for this change

### CODE REVIEW

**Comment by madan96** [comment]
Currently, we launch all the different controllers (including `group_joint_velocity_controller` for both gen2 and gen3. This will result in errors for the controllers that haven't been configured properly. Should we use conditionals until gen3 supports all of them?

File: launch/default.launch [hunk 1]
```diff
@@ -7,13 +7,27 @@
   <arg name="detector" default="false" doc="Which network to use for food perception. Set to 'false' for simulation."/>
   <arg name="version" default="2" doc="Kinova Arm Version (JACO = 2, Kortex = 3)" />
   <arg name="dof" default="6" doc="Number of degrees of freedom in arm" />
+  <arg name="ip_address" default="132.236.20.28" doc="IP address for TCP/UDP comms"/>
```
Comment (egordon) [inline launch/default.launch:10]: I would add a line break here and a docstring making clear that these are the Gen3-specific arguments.

File: launch/default.launch [hunk 2]
```diff
@@ -7,13 +7,27 @@
...
+          output="screen" if="$(eval arg('version') == 3)">
+      <param name="ip_address" value="$(arg ip_address)"/>
+      <param name="username" value="$(arg username)"/>
+      <param name="password" value="$(arg password)"/>
+      <param name="api_session_inactivity_timeout_ms" value="$(arg api_session_inactivity_timeout_ms)"/>
+      <param name="api_connection_inactivity_timeout_ms" value="$(arg api_connection_inactivity_timeout_ms)"/>
+   </node>
 
    <!-- load controllers -->
    <rosparam file="$(find libada)/config/gen$(arg version)_$(arg dof)dof.yaml" command="load" />
```
Comment (egordon) [inline launch/default.launch:33]: For right now, you can have 2 separate commands cordoned off by version.

Make sure that all controllers you have running for a given version actually work and won't throw a bunch of errors being initialized (I'm not sure if, say, you don't have an effort interface, whether you can init the effort controller stopped without erroring out).

As a note: in the future, Aikido's Robot (specifically, a RosRobot subclass) will have the ability to load the controllers directly (using the YAML as a guide), potentially removing the need for spawning them here.

**Review by egordon — CHANGES_REQUESTED** [review]
Happy to approve once the minor comments are addressed (mostly the controller loading, since this file needs to work with both Gen2 and Gen3 at all times).

**Review by egordon — APPROVED** [review]
Looks good, thank you!
FYI: Make sure to re-request review when it is ready.

**Comment by madan96** [comment]
Oh! I actually wanted to test it out one last time. I'll just test the `master` I guess.

**Pull request merged**